solutions/remotemonitoring/scripts/individual/deployment-configmap.yaml

```diff
@@ -60,6 +60,8 @@ data:
 
   # PCS Config
   config.webservice.url: "http://config-svc:9005/v1"
+  config.seedtemplate: "default"
+  config.solution.type: "{Solution type}"
```

Comment:
Are we setting this code already, through k8smanager.ts?

Reply:
That one is for diagnostics.solution.type

Reply:
Then do we still need `config.solution.type` since this is already set [here](https://github.com/Azure/pcs-cli/blob/master/src/k8smanager.ts#L202)

Reply:
I can use diagnostics.solution.type. Ideally shared variables should be under different section